### DISCUSSION TITLE
How to Set Default Terminal Directory in VS Code Workspace?

### DISCUSSION THREAD

**Discussion by holooon** [Cyfrin/foundry-full-course-cu#2230] [Q&A] [answered]
Hi,

I'm working on a project using Foundry in VS Code, and I have the following folder structure:

foundry-f23 (path: /home/#####/foundry-f23)
Inside this, there's another folder: foundry-simple-storage-f23 (path: /home/#####/foundry-f23/foundry-simple-storage-f23)
Currently, when I open the terminal in VS Code, it starts in the foundry-f23 directory by default. However, I would like the terminal to start in the foundry-simple-storage-f23 directory instead.

Is there a way to configure this so that every time I open the terminal in VS Code, it starts in the foundry-simple-storage-f23 folder by default?

Thank you!

**Comment by EngrPips** [comment, marked as answer]
Hello @holooon, You can navigate through your files to open a specific file you want. I believe this tutorial https://www.youtube.com/watch?v=u3PMR8voOo0 will help.

**Reply by holooon** [reply]
Damn, how did I missed that, thank's bro!

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
